.travis.mk

```diff
@@ -134,11 +141,14 @@ build_debian: configure_debian
 	make -j
 
 test_debian_no_deps: build_debian
+	pip install -r test-run/requirements.txt
```

Comment:
Since we split deps / test goals, I would move it to deps.

Reply:
Right, improved as suggested.